addons/l10n_sa/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,478 @@
+            <field name="sequence" eval="4"/>
+        </record>
+        <record id="tax_report_line_exempt_purchases_tax" model="account.tax.report.line">
+            <field name="name">11. Exempt Purchases (Tax)</field>
+            <field name="tag_name">11. Exempt Purchases (Tax)</field>
+            <field name="parent_id" ref="tax_report_line_vat_all_expenses_tax"/>
+            <field name="report_id" ref="tax_report_vat_filing"/>
+            <field name="code">EXM_PUR_T</field>
+            <field name="sequence" eval="5"/>
+        </record>
+        <record id="tax_report_line_net_purchases_tax" model="account.tax.report.line">
+            <field name="name">12. Net Purchases (Tax)</field>
+            <field name="parent_id" ref="tax_report_line_vat_all_expenses_tax"/>
+            <field name="report_id" ref="tax_report_vat_filing"/>
+            <field name="formula">STD_PUR_T + CUST_PUR_T + RCM_PUR_T + ZER_PUR_T + EXM_PUR_T</field>
+            <field name="sequence" eval="6"/>
+        </record>
+        <record id="tax_report_line_net_vat_due" model="account.tax.report.line">
+            <field name="name">Net VAT Due</field>
+            <field name="report_id" ref="tax_report_vat_filing"/>
+            <field name="sequence" eval="5"/>
+        </record>
+        <record id="tax_report_line_total_value_of_due_tax_for_the_period" model="account.tax.report.line">
+            <field name="name">Total value of due tax for the period</field>
+            <field name="parent_id" ref="tax_report_line_net_vat_due"/>
+            <field name="report_id" ref="tax_report_vat_filing"/>
+            <field name="formula">STD_SALE_T + SPCL_SALE_T + ZERO_SALE_T + EXP_SALE_T + EXM_SALE_T</field>
+            <field name="sequence" eval="1"/>
+        </record>
+        <record id="tax_report_line_total_value_of_recoverable_tax_for_the_period" model="account.tax.report.line">
+            <field name="name">Total value of recoverable tax for the period</field>
+            <field name="parent_id" ref="tax_report_line_net_vat_due"/>
+            <field name="report_id" ref="tax_report_vat_filing"/>
+            <field name="formula">STD_PUR_T + CUST_PUR_T + RCM_PUR_T + ZER_PUR_T + EXM_PUR_T</field>
+            <field name="sequence" eval="2"/>
+        </record>
+        <record id="tax_report_line_net_vat_due_or_reclaimed_for_the_period" model="account.tax.report.line">
+            <field name="name">Net VAT due (or reclaimed) for the period</field>
+            <field name="parent_id" ref="tax_report_line_net_vat_due"/>
+            <field name="report_id" ref="tax_report_vat_filing"/>
+            <field name="formula">STD_SALE_T + SPCL_SALE_T + ZERO_SALE_T + EXP_SALE_T + EXM_SALE_T - (STD_PUR_T + CUST_PUR_T + RCM_PUR_T + ZER_PUR_T + EXM_PUR_T)</field>
+            <field name="sequence" eval="3"/>
+        </record>
+        <record id="tax_report_withholding_tax" model="account.tax.report">
```

Comment:
separator (comment that indicates we are doing a different report) or different files

Reply:
added a comment and some break lines to show the difference